src/main/java/net/openhft/chronicle/bytes/internal/UnsafeText.java

```diff
@@ -0,0 +1,308 @@
+/*
+ * Copyright (c) 2016-2022 chronicle.software
+ *
+ *     https://chronicle.software
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package net.openhft.chronicle.bytes.internal;
+
+import net.openhft.chronicle.core.*;
+
+import static net.openhft.chronicle.core.UnsafeMemory.MEMORY;
+import static net.openhft.chronicle.core.UnsafeMemory.UNSAFE;
+
+/**
+ * These are fast, unsafe ways to render text.
+ * NOTE: The caller has to ensure there is always plenty of memory to perform this operation.
+ */
+public final class UnsafeText {
```

Comment:
Hi Peter is this a brand new class or refactored from something else?

Reply:
Moved from Core to Bytes to make it internal to Bytes, and eventually phase it out.